bugswarm/analyzer/entry.py

```diff
@@ -90,10 +80,10 @@ def print_usage():
     log.info('-----------------------------------------------------------------------')
     log.info('To analyze compare reproduced log with original Travis log:')
     log.info('    Python')
-    log.info('      Example: python3 entry.py -r 45123523.log -o 45123523-orig.log -j 45123523')
+    log.info('      Example: python3 entry.py -l 45123523.log -o 45123523-orig.log -j 45123523')
     log.info('    Java')
-    log.info('      Example: python3 entry.py -r 308408479.log -o 308408479-orig.log -j 308408479 -b maven')
-    log.info('      Example: python3 entry.py -r 308408479.log -o 308408479-orig.log -j 308408479 -t 67bc28d1a4fe777b50'
+    log.info('      Example: python3 entry.py -l 308408479.log -o 308408479-orig.log -j 308408479 -b maven')
+    log.info('      Example: python3 entry.py -l 308408479.log -o 308408479-orig.log -j 308408479 -t 67bc28d1a4fe777b50'
```

Comment:
Why are we changing `-r` to `-l`?

Reply:
Error on my end, assumed that since the other examples were utilizing `-l` that this was an error and thought that `-r` represented repo.